core/models/asset.py

```diff
@@ -149,7 +149,11 @@ class AssetType(models.Model):
     asset_type = models.CharField(unique=True, max_length=50)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified = models.DateTimeField(auto_now=True, editable=False)
-    asset_sub_category = models.ForeignKey(AssetSubCategory, on_delete=models.PROTECT)
+    asset_sub_category = models.ForeignKey(
+        AssetSubCategory,
+        on_delete=models.PROTECT
+    )
+    has_specs = models.NullBooleanField(blank=False, null=True)
```

Comment:
sorry, let's have a `default=False` and do away with `core/migrations/populate_specs_field.py`

Reply:
also delete `core/migrations/0001_auto_20181130_1125.py` and `core/migrations/0033_auto_20181121_2302.py`, then makemigrations again to have just one file

Reply:
Done!